resources/wikia/libraries/headroom/headroom.css

```diff
@@ -0,0 +1,10 @@
+.headroom {
+	will-change: transform;
+	transition: transform 200ms linear;
+}
+.headroom--pinned {
+	transform: translateY(0%);
+}
+.headroom--unpinned {
```

Comment:
I believe we should avoid formatting library code to make it easier to update it in future.